client/ayon_kitsu/hooks/pre_status_cahnge.py

```diff
@@ -0,0 +1,124 @@
+from ayon_applications import PreLaunchHook
+from ayon_kitsu.addon import is_kitsu_enabled_in_settings
+
+import os
+import gazu
+import re
+
+
+
+class PreStatusChange(PreLaunchHook):
```

Comment:
I think it'd be better to make it a post launch hook.
```suggestion
class PreStatusChange(PostLaunchHook):
```